VideoLocker/build.gradle

```diff
@@ -170,8 +170,9 @@ android {
         // minimum version is Android 4.0
         minSdkVersion 14
         targetSdkVersion 21
+
         versionCode 77
```

Comment:
Shouldn't the version code be incremented as well, to properly set up for the upgrade?